.github/workflows/release.yaml

```diff
@@ -110,6 +110,11 @@ jobs:
           mac_certs_password: ${{ secrets.MAC_PASSWORD }}
         env:
           EP_PRE_RELEASE: true
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+          SENTRY_ORG: "nodefactory"
+          SENTRY_PROJECT: "chainguardian"
+          SENTRY_SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
```

Comment:
this is not used, can be removed

Reply:
it's used by webpack during release process to upload sourcemap